includes/attendee/attendee-repository.php

```diff
@@ -86,47 +86,77 @@ public function remove_attendee( int $event_id, int $user_id ): void {
 	/**
 	 * @throws Exception
 	 */
-	public function get_attendee( int $event_id, int $user_id ): ?Attendee {
+	public function get_attendee_for_event_for_user( int $event_id, int $user_id ): ?Attendee {
+		$attendees = $this->get_attendees_for_events_for_user( array( $event_id ), $user_id );
+		if ( empty( $attendees ) ) {
+			return null;
+		}
+		return $attendees[0];
+	}
+
+	/**
+	 * @var int[] $event_ids
+	 * @return Attendee[]
+	 * @throws Exception
+	 */
+	public function get_attendees_for_events_for_user( array $event_ids, int $user_id ): array {
```

Comment:
I have no idea what we can rename this function to but it may be prone to typo errors when considering the preceding function in line 89 `public function get_attendee_for_event_for_user( int $event_id, int $user_id )` even though they serve different purposes.

Reply:
Maybe we can rename the other one to `get_single_attendee()`.